babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -554,6 +554,47 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testCurrentTimeStampFunction() {
+    final String sql = "select current_timestamp";
+    final String expected = "SELECT CURRENT_TIMESTAMP";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testCurrentTimeFunction() {
+    final String sql = "select current_time";
+    final String expected = "SELECT CURRENT_TIME";
+    sql(sql).ok(expected);
+  }
+  @Test public void testBothTimeFunctions() {
+    final String sql = "select current_time, current_timestamp";
+    final String expected = "SELECT CURRENT_TIME, CURRENT_TIMESTAMP";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testCurrentTimeStampFunctionBigQuery() {
```

Comment:
Would it be valid to pass in CURRENT_TIME() with parentheses and see if it unparses to the same thing? I'm not sure if the parsing logic is there to parse the parentheses or it's just supposed to unparse into BigQuery.

Reply:
Looks like it works, I'll add additional tests to cover this.